common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -75,9 +73,15 @@ def check_course_equality(course1, course2):
     def _assert_user_present(self, user, present=True):
         """ Checks if specified user present on Course Team page """
         if present:
-            self.assertIn(user.get('username'), self.page.usernames)
+            self.page.wait_for(
```

Comment:
Note to reviewers: two different test points sporadically failed with self.page.usernames = [] (username wasn't in it yet). The impacted test points passed 30 times in Jenkins with this change.